auth/src/main/java/com/firebase/ui/auth/ui/phone/VerifyPhoneNumberFragment.java

```diff
@@ -219,6 +222,14 @@ private String getPseudoValidPhoneNumber() {
     }
 
     private void setupCountrySpinner() {
+        Bundle params = getArguments().getBundle(ExtraConstants.PARAMS);
+        if (params != null) {
+            mCountryListSpinner.setWhitelistedCountryCodes(
+                    params.getStringArrayList(ExtraConstants.WHITELISTED_COUNTRY_CODES));
```

Comment:
What's the default for `getStringArrayList`, is it `null`?

Reply:
Yes, it returns null if the key isn't present.